.circleci/config.yml

```diff
@@ -469,23 +415,10 @@ workflows:
   build_and_test:
     unless: << pipeline.parameters.prerelease_commit_sha >>
     jobs:
-      - setup:
-          filters:
-            branches:
-              ignore:
-                - builds/facebook-www
```

Comment:
@kassens didn't look too deeply into where this should now go, but could you disable any CircleCI workflows on the `builds/facebook-www` branch? Nothing will break but there will be nothing to run on that branch since it only contains a compiled folder with no tests or anything. What I did before was just disable the main `setup` job that kicked off everything, so if `build_and_test` is the new parent you could add back the ignore here

Reply:
I see how this was working. I can update this.